config/locales/fr.yml

```diff
@@ -817,6 +858,7 @@ fr:
             comments: Commentaires
             created_at: Soumise le
             endorsements: Soutiens
+            first_interacted_at: First interacted at
```

Comment:
🛑 Please directly add French translation ... Adding missing keys with non-translated values will provoke more issues related to translations from client ...